virttest/utils_test/libvirt.py

```diff
@@ -3677,3 +3678,56 @@ def check_dumpxml(vm, content, err_ignore=False):
                 raise exceptions.TestFail("Expected '%s' was not found in "
                                           "%s's xml" % (content, vm.name))
     return True
+
+
+# pylint: disable=E1121
+def check_domjobinfo(vm, params, option="", remote_virsh_dargs=None):
```

Comment:
suggest create one separate utils file such as utils_domjob.py under avocado-vt/virttest to avoid putting everything into libvirt.py file

Reply:
@chunfuwen If we want a separate file, I prefer libvirt_domjobinfo.py under avocado-vt/virttest/utils_test. What do you think?

Reply:
sound ok

Reply:
+1